HTMLSerializer.js

```diff
@@ -112,6 +126,30 @@ var HTMLSerializer = class {
   }
 
   /**
+   * Takes an html node of type Node.TEXT_NODE, and add its text content with
+   *     all characters properly escaped to |this.html|.
+   * @param {Node} node The text node.
+   */
+   // TODO(sfine): Take care of attribute value normalization:
+   // https://developers.whatwg.org/the-iframe-element.html#the-iframe-element
+  processText(node) {
+    var win = node.ownerDocument.defaultView;
+    var windowDepth = this.windowDepth(win);
+    var text = node.textContent;
+    // & must be processed first.
+    text = text.replace(/&/g, this.escapedCharacter('&', windowDepth+1));
+    var chars = Object.keys(this.escapedCharacters);
+    for (var i = 0, char; char = chars[i]; i++) {
```

Comment:
You can use "for (var char in this.escapedCharacters)" to avoid the 'i' and 'chars' vars.